manifests/repo.pp

```diff
@@ -188,7 +188,7 @@
         logoutput => on_failure,
       }
 
-      if $::mrepo_env {
+      if $mrepo_env {
```

Comment:
Yeah, `mrepo_logging` appears to be a file location for when the cron/exec runs, not super clean. Should probably default to /var/log/mrepo.sync.log or something